src/Invictus.Testing/LogicAppClient.cs

```diff
@@ -0,0 +1,498 @@
+            {
+                foreach (KeyValuePair<string, string> header in headers)
+                {
+                    request.Headers.Add(header.Key, header.Value);
+                }
+
+                await HttpClient.SendAsync(request);
+            }
+        }
+
+        /// <summary>
+        /// Gets the URL on which the workflow with trigger can be run by searching the workflow for configured triggers.
+        /// </summary>
+        /// <exception cref="LogicAppTriggerNotFoundException">When no trigger can be found on the logic app.</exception>
+        public async Task<LogicAppTriggerUrl> GetTriggerUrlAsync()
+        {
+            string triggerName = await GetTriggerNameAsync();
+            LogicAppTriggerUrl triggerUrl = await GetTriggerUrlByNameAsync(triggerName);
+            
+            return triggerUrl;
+        }
+
+        /// <summary>
+        /// Gets the URL on which the workflow with the <paramref name="triggerName"/> can be run.
+        /// </summary>
+        /// <param name="triggerName">The name of the trigger that relates to a workflow.</param>
+        public async Task<LogicAppTriggerUrl> GetTriggerUrlByNameAsync(string triggerName)
+        {
+            Guard.NotNullOrEmpty(triggerName, nameof(triggerName));
+            
+            _logger.LogTrace("Request the workflow trigger URL of logic app '{LogicAppName}' in resource group '{ResourceGroup}'", _logicAppName, _resourceGroup);
+            WorkflowTriggerCallbackUrl callbackUrl = 
+                await _logicManagementClient.WorkflowTriggers.ListCallbackUrlAsync(_resourceGroup, _logicAppName, triggerName);
+
+            return new LogicAppTriggerUrl
+            {
+                Value = callbackUrl.Value, 
+                Method = callbackUrl.Method
+            };
+        }
+
+        private async Task<string> GetTriggerNameAsync()
+        {
+            IPage<WorkflowTrigger> triggers = await _logicManagementClient.WorkflowTriggers.ListAsync(_resourceGroup, _logicAppName);
+            
+            if (triggers.Any())
+            {
+                return triggers.First().Name;
+            }
+
+            throw new LogicAppTriggerNotFoundException(
+                $"Cannot find any trigger for logic app '{_logicAppName}' in resource group '{_resourceGroup}'");
+        }
+
+        /// <summary>
+        /// Temporary enables a static result for an action with the given <paramref name="actionName"/> on the logic app,
+        /// and disables the static result when the returned instance gets disposed.
+        /// </summary>
+        /// <param name="actionName">The name of the action to enable the static result.</param>
+        /// <returns>
+        ///     An instance to control when the static result for the action on the logic app should be disabled.
+        /// </returns>
+        public async Task<IAsyncDisposable> TemporaryEnableSuccessStaticResult(string actionName)
```

Comment:
Can we add a test for this?

Reply:
See [this conv above](https://github.com/invictus-integration/testing-framework/pull/27#discussion_r435718037)

Reply:
Done.